app/views/admin/settings/forms/_images.html.erb

```diff
@@ -54,8 +54,8 @@
                           rows: 6,
                           placeholder: Constants::Settings::General::DETAILS[:logo_svg][:placeholder] %>
           <% if Settings::General.logo_svg.present? %>
-            <div class="site-logo">
-              <%= Settings::General.logo_svg.html_safe %>
+            <div class="mt-2">
+              <a href="<%= Settings::General.logo_svg.html_safe %>" target="_blank" rel="noopener"><img src="<%= Settings::General.logo_svg.html_safe %>" alt="<%= community_name %> logo" style="max-width: 100%" loading="lazy" /></a>
```

Comment:
I added an SVG logo during setup, but I can't see it in this section 🤔  Maybe I'm looking in the wrong place (I'm looking in Admin > Customization > Images)?

![screenshot of the mentioned admin page, with no images successfully loaded](https://user-images.githubusercontent.com/20773163/144202170-93eaba97-6b60-41c4-bcf5-65422ee25023.png)



```suggestion
              <a href="<%= Settings::General.logo_svg.html_safe %>" target="_blank" rel="noopener"><img src="<%= Settings::General.logo_svg.html_safe %>" alt="preview selected logo" style="max-width: 100%" loading="lazy" /></a>
```

(suggestion / not blocking) - it would be good for the accessible name to better reflect the link destination I think

Reply:
> I added an SVG logo during setup, but I can't see it in this section 🤔 Maybe I'm looking in the wrong place

@aitchiss this is expected 😅. You should only see it in the header. There is currently no way to update the logo after you leave the Creator Settings Page. 

I mention in the description of this PR that we will tackle it in a subsequent PR. 
i.e. 

```
In a subsequent PR we will tackle:
......
Create an upload component in the admin to allow the creator to update their logo.
```

In this case it's okay and encouraged to implement the feature incrementally since it's behind a feature flag.

Reply:
That's cool, I just wasn't sure where I was able to check this code change 🙃

Reply:
no worries! sounds good 🙂